files/grolemund-2016-09.md

```diff
@@ -0,0 +1,65 @@
+
+*   How to recognize expert blindspot (i.e., relying on knowledge that the student does not yet have).
+
+*   How to give feedback (practiced with video recording and feedback).
+
+*   How to facilitate participation withan Etherpad or​Google Doc.
+
+## Positive
+
+*   Tries to anchor practice in educational psychology.
+
+*   Introduces the idea of peer-led instruction.
+
+*   Course material is open source, which lets many instructors work together to refine it.
+
+*   Lessons have a useful infrastructure:
+    *   Workshop webpage
+    *   Online course notes and exercises
+￼
+## Negative
+
+*   Includes some material that is not useful, and does not bring material together in a way that can be quickly applied in a classroom.
+
+*   Provides an incomplete picture of what it takes to build a mental model because it does not connect working memory to cognitive load theory or mention a main extension of Cognitive Load Theory (Multimedia Learning Theory):
+    *   Learning tasks involve three types of cognitive load: inherent, generative, and external
+    *   What's missing: a person only has a finite amount of cognitive load to expend at any given moment. Inherent load exceeds this bandwidth, hence you need to scaffold the presentation of complex material. The student must choose to expend generative load, hence you need to show the student how to expend generative load in an efficient way and then motivate them to expend it. Extraneous load diminishes this bandwidth, hence you need to reduce distractions.
+
+*   Multimedia learning theory (not covered)
+    *   Available load is distributed across visual and verbal information channels.
+    *   Material is easier to learn when split simultaneously across different formats.
+    *   It is more difficult to learn when multiple types of information are presented simultaneously in the same format.
+    *   This presents a clear case for visual slides and insights on how to avoid bad slides.
+
+*   Due to the above, SC does not use slides when they would be appropriate, which makes their own material difficult to learn (especially in the instructor workshop).
+    *   Along with the above, SC does not discuss how to make effective slides, when to use them, and how to interleave them with live coding
+
+*   Does not discuss how to keep energy levels and student participation high in a workshop.
+
+*   Does not provide a clear view of what peer-led instruction would look like in a workshop.
+
+*   Training does not cover solutions to common logistical problems: installation issues, multiple operating systems, ensuring prerequisite knowledge, disseminating materials, etc.
+
+*   Spends a lot of time on live teaching exercises that are too artificial to be helpful.
+    *   Teachers do not get to use projectors, slides, or computers (to code) during the exercises, which makes them de facto communication style evaluations.
+    *   This is useful, but justifies less time.
```

Comment:
I think the issue here is understanding and emphasising what the focus of live teaching (video) exercises is. My understanding is that it actually is:
* learning how to give and receive feedback;
* learning what happens to our body language and voice when we teach (present);
* getting the experience (in many cases, the first time!) of live coding teaching (speaking whilst typing etc);
* seeing how the way we present/teach changes when we go from loosely defined topic (first video) through more concrete contents (2nd video) to some pre-prepared contents.

The focus of these live exercises is **not** on getting our trainees into perfect SWC/DC or other teachers. I dare to say that this is impossible due to:
- you can't do it in 2 days - becoming a good instructor takes several workshops;
- there is no *perfect* way of teaching our materials (or any other material, I think).

The last point is particularily important. I have been asked a couple of times at Instructor Training "so what is the perfect way of teaching this"? This question made me understand that the way I was delivering this particular Instructor Training module had incorrect focus. My trainees thought I was trying to turn them into perfect instructors. But in fact what we do during IT is learning how to do * jugyokenkyu* :-)